tests/contenttypes_tests/test_models.py

```diff
@@ -99,6 +99,24 @@ def test_get_for_models_migrations(self):
             cts, {ContentType: ContentType.objects.get_for_model(ContentType)}
         )
 
+    def test_get_for_models_migrations_create_model(self):
```

Comment:
You'll want to use `@isolate_apps("contenttypes_tests")` here to avoid having `Foo` pollute the global registry and leak through the suite.

Reply:
Thank you ⭐